packages/storybook/stories/CallComposite/snippets/Container.snippet.tsx

```diff
@@ -23,15 +23,16 @@ export type ContainerProps = {
 };
 
 const isTeamsMeetingLink = (link: string): boolean => link.startsWith('https://teams.microsoft.com/l/meetup-join');
-const isGroupLink = (link: string): boolean => link.indexOf('-') !== -1;
+const uuidRegexExp = /^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}$/gi;
```

Comment:
Looks like we have "uuid" dependency in this folder, can we do 

import uuid from 'uuid';
...
uuid.validate(str);